pkg/plugin/processor/builtin/impl/avro/schemaregistry/fake.go

```diff
@@ -94,10 +94,10 @@ const (
 	errorCodeSchemaNotFound  = 40403
 )
 
-// fakeRegistry is a simple fake registry meant to be used in tests. It stores
+// FakeRegistry is a simple fake registry meant to be used in tests. It stores
```

Comment:
I think this might also be more usfeul in conduit-commons, I expect we'll need it in tests in the connector and processor SDK.

Reply:
Hmm, but the Connector SDK for example should use a schema registry only through Conduit, i.e. it's Conduit's schema API which we can mock and use in the tests.

Reply:
Mocking the registry is a valid approach, but the registry is already a pretty dumb thing, when I was working on that `schemaregistry` package I found it much simpler to work with the fake registry (that's why I added it in the first place). Similarly how we use the in-memory DB in Conduit tests, I think it will be easier to write tests using a proper in-memory schema registry.

Although if you prefer mocking, I won't stand in your way 😉

Reply:
It will call the gRPC service, that's right, I'm saying that in tests we might want to forward the calls to this ~fake~ in-memory registry instead of a mock. In other words, instead of a mock we'd have a small adapter type, which implements the gRPC service interface that the connector is using and calls the in-memory registry.

Reply:
I think you're making a good, but at least in this moment, I'm not quite convinced it's worth. So maybe we just leave the FakeRegistry, and move it into commons when/if we realize it's easier to test with it? That way, we'll hopefully keep the commons repo a bit cleaner.